tests/python/test_offset.py

```diff
@@ -0,0 +1,32 @@
+import taichi as ti
+
+
+@ti.all_archs
+def test_accessor():
+    a = ti.var(dt=ti.i32)
+
+    ti.root.dense(ti.ijkl, 128).place(a, offset=(1024, 2048, 2100, 2200))
+
+    a[1029, 2100, 2200, 2300] = 1
+    assert a[1029, 2100, 2200, 2300] == 1
+
+
+@ti.all_archs
+def test_struct_for():
+    a = ti.var(dt=ti.i32)
+
+    offset = 1024, 2048, 2100, 2200
```

Comment:
Shall we add tests about negative offsets?

Reply:
Good idea. Added! Waiting for CI to pass.